properties.schema

```diff
@@ -15,15 +15,15 @@
     "pageLevelProgressIndicatorBar": {
       "type": "string",
       "required": true,
-      "default": "Progress bar. Select here to view your current progress, and select an item to navigate to it. You have completed ",
+      "default": "Page progress. Use this to listen to the list of regions in this topic and whether they're completed. You can jump directly to any that are incomplete or which sound particularly interesting. {{percentageComplete}}%",
```

Comment:
There's a problem with including `{{percentageComplete}}` here in that it'll only update when you navigate away from the page and back again. It also didn't seem to be displaying anything initially so you'd just get "[..] which sound particularly interesting. %".

Reply:
is this fixed now?